hack/jenkins/test-flake-chart/report_flakes.sh

```diff
@@ -85,6 +85,10 @@ awk -F, 'NR>1 {
   | sort -g -t, -k2,2 \
   >> "$TMP_FAILED_RATES"
 
+# Filter out arm64 and macOS tests until they're more stable
+grep -v arm64 "$TMP_FAILED_RATES" > "$TMP_FAILED_RATES"
+grep -v macOS "$TMP_FAILED_RATES" > "$TMP_FAILED_RATES"
```

Comment:
Could do grep -v "arm64\|macOS" to reduce some duplication but not required

Reply:
yeah, that's definitely true, I just wanted to make it super clear and allow for easy deletion.